manifests/config.pp

```diff
@@ -115,8 +115,13 @@
   $pool_test_while_idle = pick($jira::pool_test_while_idle, true)
   $pool_test_on_borrow = pick($jira::pool_test_on_borrow, false)
 
-  # This is just for consistency
-  $connection_settings = $jira::connection_settings
+  # JIRA will complain if these aren't set for PostgreSQL (will work fine though)
+  # https://confluence.atlassian.com/jirakb/connection-problems-to-postgresql-result-in-stuck-threads-in-jira-1047534091.html
+  if $jira::db == 'postgresql' {
```

Comment:
personally, I prefer a selector statement, but if/else works as well.

Reply:
I prefer a simple if / else until there are more choices; I can never remember the selector syntax and it feels very clunky with only one option.